doc/whats-new.rst

```diff
@@ -39,6 +39,9 @@ Documentation
   By `Justus Magin <https://github.com/keewis>`_.
 - start a list of external I/O integrating with ``xarray`` (:issue:`683`, :pull:`4566`).
   By `Justus Magin <https://github.com/keewis>`_.
+- add concat examples and improve combining documentation (:issue:`4620`, :pull:`4645`).
+  By `Ray Bell <https://github.com/raybellwaves>`_ and
+  `Justus Magin <https://github.com/keewis>`_.
```

Comment:
you don't really have to add me as co-author, I only reviewed and applied a tool (you can keep it as is if you want, though)

Reply:
it's fine